src/api-engine/api/lib/configtxgen/configtxgen.py

```diff
@@ -8,7 +8,7 @@
 class ConfigTxGen:
     """Class represents cryptotxgen."""
 
-    def __init__(self, network, filepath=CELLO_HOME, configtxgen=FABRIC_TOOL, version="2.2.0"):
```

Comment:
Could we use some variables, instead of fixed number?
This way is more convenient to update in future.

Reply:
In my latest commit, I've used FABRIC_VERSION in _<src/api-engine/api/config.py>_ to control version number.